eo-maven-plugin/src/main/java/org/eolang/maven/ChRemote.java

```diff
@@ -77,41 +82,31 @@ public String value() {
 
     /**
      * Load all hashes and tags.
+     *
      * @return Map of them (hash -> tag)
      */
-    private static Map<String, String> safeLoad() {
-        Map<String, String> map;
+    private static String safeLoad() {
+        String cache;
         try {
-            map = ChRemote.load();
+            cache = new UncheckedText(ChRemote.load()).asString();
         } catch (final IOException ex) {
             Logger.warn(
                 ChRemote.class,
                 "Failed to load catalog of Git hashes from %s, because of %s: '%s'",
                 ChRemote.HOME, ex.getClass().getSimpleName(), ex.getMessage()
             );
-            map = new HashMap<>(0);
+            cache = "";
         }
-        return map;
+        return cache;
     }
 
     /**
      * Load all hashes and tags.
+     *
      * @return Map of them (hash -> tag)
      * @throws IOException if fails
-     * @todo #1174:90m The code duplication with ChText. It's better to reuse the logic
-     *  of parsing text source from ChText class. Maybe we can implement caching mechanism
-     *  in a way to avoid using map directly.
      */
-    private static Map<String, String> load() throws IOException {
-        final InputStream ins = new URL(ChRemote.HOME).openStream();
-        try (Scanner scanner = new Scanner(ins)) {
-            final Map<String, String> map = new HashMap<>(0);
-            while (scanner.hasNextLine()) {
-                final String line = scanner.nextLine();
-                final String[] parts = line.split("\t");
-                map.put(parts[1], parts[0]);
-            }
-            return map;
-        }
+    private static Text load() throws IOException {
+        return new TextOf(new URL(ChRemote.HOME).openStream());
```

Comment:
@volodya-lombrozo it's a bad practice to use `openStream()` and NOT close it ever. That's why I'm suggesting using `new TextOf(new InputOf(ChRemote.HOME))` -- this is safe and looks safe.

Reply:
@yegor256 Done